app/src/main/java/com/github/libretube/services/AbstractPlayerService.kt

```diff
@@ -409,6 +409,8 @@ abstract class AbstractPlayerService : MediaLibraryService(), MediaLibrarySessio
                 mediaLibrarySession = null
             }
 
+            PlayingQueue.clearCurrent()
```

Comment:
Shouldn't we clear the full queue instead here by calling `PlayingQueue#clear` and additionally change the `clear` method to null the `currentStream`?

Reply:
Probably, is there any reason why we're not doing this already? Or just that the queue and stream is reset anyway when a new player is created?

Reply:
> is there any reason why we're not clearing this already? Or just that the queue and stream is reset anyway when a new player is created?
>
There's no reason for that, we should always have cleared the `currentStream`. It's just been overlooked because we resetted it when a new player is created, so it's not been causing any obvious issues before the player service refactorings a few months ago, yeah.